environment.yml

```diff
@@ -1,319 +1,9 @@
-  - libedit=3.1.20191231=h1de35cc_1
-  - libffi=3.3=hb1e8313_2
-  - libgfortran=3.0.1=h93005f0_2
-  - libiconv=1.16=h1de35cc_0
-  - liblief=0.10.1=h0a44026_0
-  - libllvm9=9.0.1=h21ff451_1
-  - libpng=1.6.37=ha441bb4_0
-  - libsodium=1.0.18=h1de35cc_0
-  - libspatialindex=1.9.3=h0a44026_0
-  - libssh2=1.9.0=ha12b0ac_1
-  - libtiff=4.1.0=hcb84e12_1
-  - libxml2=2.9.10=h3b9e6c8_1
-  - libxslt=1.1.34=h83b36ba_0
-  - llvm-openmp=10.0.0=h28b9765_0
-  - llvmlite=0.33.0=py37ha11be7d_1
-  - locket=0.2.0=py37_1
-  - lxml=4.5.2=py37h63b7cb6_0
-  - lz4-c=1.9.2=h0a44026_0
-  - lzo=2.10=h1de35cc_2
-  - markupsafe=1.1.1=py37h1de35cc_0
-  - matplotlib=3.2.2=0
-  - matplotlib-base=3.2.2=py37h5670ca0_0
-  - mccabe=0.6.1=py37_1
-  - mistune=0.8.4=py37h1de35cc_0
-  - mkl=2019.4=233
-  - mkl-service=2.3.0=py37hfbe908c_0
-  - mkl_fft=1.1.0=py37hc64f4ea_0
-  - mkl_random=1.1.1=py37h959d312_0
-  - mock=4.0.2=py_0
-  - more-itertools=8.4.0=py_0
-  - mpc=1.1.0=h6ef4df4_1
-  - mpfr=4.0.2=h9066e36_1
-  - mpmath=1.1.0=py37_0
-  - msgpack-python=1.0.0=py37h04f5b5a_1
-  - multipledispatch=0.6.0=py37_0
-  - nbconvert=5.6.1=py37_0
-  - nbformat=5.0.7=py_0
-  - ncurses=6.2=h0a44026_1
-  - networkx=2.4=py_1
-  - ninja=1.10.1=py37h879752b_0
-  - nltk=3.5=py_0
-  - nose=1.3.7=py37_2
-  - notebook=6.0.3=py37_0
-  - numba=0.50.1=py37h959d312_1
-  - numexpr=2.7.1=py37hce01a72_0
-  - numpy=1.18.5=py37h1da2735_0
-  - numpy-base=1.18.5=py37h3304bdc_0
-  - numpydoc=1.1.0=py_0
-  - olefile=0.46=py37_0
-  - openpyxl=3.0.4=py_0
-  - openssl=1.1.1g=h1de35cc_0
-  - packaging=20.4=py_0
-  - pandas=1.0.5=py37h959d312_0
-  - pandoc=2.10=0
-  - pandocfilters=1.4.2=py37_1
-  - parso=0.7.0=py_0
-  - partd=1.1.0=py_0
-  - path=13.1.0=py37_0
-  - path.py=12.4.0=0
-  - pathlib2=2.3.5=py37_0
-  - pathtools=0.1.2=py_1
-  - patsy=0.5.1=py37_0
-  - pcre=8.44=hb1e8313_0
-  - pep8=1.7.1=py37_0
-  - pexpect=4.8.0=py37_0
-  - pickleshare=0.7.5=py37_0
-  - pillow=7.2.0=py37ha54b6ba_0
-  - pip=20.1.1=py37_1
-  - pkginfo=1.5.0.1=py37_0
-  - pluggy=0.13.1=py37_0
-  - ply=3.11=py37_0
-  - prometheus_client=0.8.0=py_0
-  - prompt-toolkit=3.0.5=py_0
-  - prompt_toolkit=3.0.5=0
-  - psutil=5.7.0=py37h1de35cc_0
-  - ptyprocess=0.6.0=py37_0
-  - py=1.9.0=py_0
-  - py-lief=0.10.1=py37haf313ee_0
-  - pycodestyle=2.6.0=py_0
-  - pycosat=0.6.3=py37h1de35cc_0
-  - pycparser=2.20=py_2
-  - pycrypto=2.6.1=py37haf1e3a3_10
-  - pycurl=7.43.0.5=py37ha12b0ac_0
-  - pydocstyle=5.0.2=py_0
-  - pyflakes=2.2.0=py_0
-  - pygments=2.6.1=py_0
-  - pylint=2.5.3=py37_0
-  - pyodbc=4.0.30=py37h0a44026_0
-  - pyopenssl=19.1.0=py_1
-  - pyparsing=2.4.7=py_0
-  - pyqt=5.9.2=py37h655552a_2
-  - pyrsistent=0.16.0=py37h1de35cc_0
-  - pysocks=1.7.1=py37_0
-  - pytables=3.6.1=py37h5bccee9_0
-  - pytest=5.4.3=py37_0
-  - python=3.7.7=hf48f09d_4
-  - python-dateutil=2.8.1=py_0
-  - python-jsonrpc-server=0.3.4=py_1
-  - python-language-server=0.34.1=py37_0
-  - python-libarchive-c=2.9=py_0
-  - python.app=2=py37_10
-  - pytorch=1.6.0=py3.7_0
```

Comment:
Also best to avoid the specific python version (the `py3.7_0` part) and let conda figure that out